.github/workflows/release.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32 # pin@5.0.2
         with:
-          go-version: '1.22'
+          go-version-file: 'go.mod'
```

Comment:
In the current setup, the most recent version of `1.22` will always be used. This change will cause it to always us the specific patch version specified in the `go.mod` file. That means, we'll need a PR to bump the Go version every time there is a new Go release and it will need to be cherry picked to the release branch.

This will add more work. As the person who does most of this work, the automation just handling this without me needing to do the work is nice. Why should we give that up?

Note, `go 1.22.0` sets the minimum version to be used by the Go compiler. It can use newer versions. When you tell the `setup-go` action to use the `go.mod` file it will use the exact patch version specified when the patch version is specified.

Reply:
@mattfarina thanks so much. make sense. I will close this PR.